automation/check-patch.e2e-kubemacpool-functests.sh

```diff
@@ -28,9 +29,46 @@ main() {
     echo "Deploy KubeVirt latest stable release"
     ./hack/deploy-kubevirt.sh
 
+    echo "Get skip tests regex"
+    SKIPPED_TESTS=$(__get_skipped_tests)
+
     # Run KubeMacPool functional tests
     cd ${TMP_COMPONENT_PATH}
-    KUBECONFIG=${KUBECONFIG} E2E_TEST_ARGS="--junit-output=$ARTIFACTS/junit.functest.xml" make functest
+
+		echo SKIPPED_TESTS=$SKIPPED_TESTS
+    KUBECONFIG=${KUBECONFIG} E2E_TEST_ARGS="--junit-output=$ARTIFACTS/junit.functest.xml --ginkgo.skip $SKIPPED_TESTS" make functest
+}
+
+function __get_skipped_tests() {
+	local skipped_regex=""
+
+	if [ "$(__is_opt_out)" == "true" ]; then
+		skipped_regex="${skipped_regex} \(opt-in\smode\)"
+	elif [ "$(__is_opt_in)" == "true" ]; then
+		skipped_regex="${skipped_regex} \(opt-out\smode\)"
+	fi
+
+	echo $skipped_regex
+}
+
+function __is_opt_in() {
+	if [ $(yaml-utils::read_multiple_docs data/kubemacpool/kubemacpool.yaml webhooks[1].namespaceSelector.matchExpressions[2].key) == 'mutatevirtualmachines.kubemacpool.io' ] && \
+	   [ $(yaml-utils::read_multiple_docs data/kubemacpool/kubemacpool.yaml webhooks[1].namespaceSelector.matchExpressions[2].operator) == 'In' ] && \
+	   [ $(yaml-utils::read_multiple_docs data/kubemacpool/kubemacpool.yaml webhooks[1].namespaceSelector.matchExpressions[2].values[0]) == 'allocate' ]; then
+	     	echo "true"
+	     	return
+	fi
+	echo "false"
+}
```

Comment:
What about just greping https://github.com/kubevirt/cluster-network-addons-operator/blob/master/hack/components/bump-kubemacpool.sh#L92 ?

`grep opt_in hack/components/bump-kubemacpool` so we don't need to know how it's done.

Reply:
I like it! :)